pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -345,6 +347,11 @@ func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.Persiste
 		AccessModes:                   claim.Spec.AccessModes,
 		PersistentVolumeReclaimPolicy: api.PersistentVolumeReclaimDelete,
 		CloudTags:                     &tags,
+		ClusterName:                   controller.clusterName,
+	}
+
+	if pv != nil {
+		volumeOptions.PVName = pv.Name
```

Comment:
In Provisioners of individual volume plugins, currently as `PV.GenerateName = "pv-cinder-"`. (pv-aws-, pv-gce-)

E.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/cinder/cinder.go#L434